spatialpandas/io/parquet.py

```diff
@@ -332,13 +339,20 @@ def _perform_read_parquet_dask(
         dataset_pieces = sorted(fragments, key=lambda piece: natural_sort_key(piece.path))
         pieces.extend(dataset_pieces)
 
+    try:
+        from dask.dataframe.utils import pyarrow_strings_enabled
```

Comment:
It could be nice with one test with this import so we are sure it does not disappear.